### PR TITLE
Add documentation for Select

### PR DESCRIPTION
**Pull request by AnnaRybkina**
React and Vue documentation for the Select.

image

### CODE REVIEW

File: docs/select/Example.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<script setup>
+  import { wSelect } from '@warp-ds/vue';
+</script>
+
+<template>
+  <h3>Standard with hint</h3>
+  <div class="component">
+    <w-select label="A label" hint="A hint" v-model="model" >
+      <option disabled selected value="">Pick something</option>
```
Comment (BalbinaK) [inline docs/select/Example.vue:9]: Was it intended to display the first option as disabled in each example? 😉

File: docs/select/react.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+## Labeling
+
+A visual label should be provided for the Select using the `label` prop.
+
+
+### Optional
+
+Add the optional prop to indicate that the select is not required.
```
Comment (BalbinaK) [inline docs/select/react.md]: minor thing but thought we could write props like this: "Add the `optional` prop"

File: docs/select/vue.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+## Import
+
+> Use in entire app
+```js
+import { Forms } from '@fabric-ds/vue'
```
Comment (BalbinaK) [inline docs/select/vue.md]: @warp-ds/vue

File: docs/select/vue.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+app.use(Forms)
+```
+
+> Use in one component and special imports
+```js
+import { fSelect } from '@fabric-ds/vue'
```
Comment (BalbinaK) [inline docs/select/vue.md]: same here

File: docs/select/vue.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+```html
+<w-select v-model="model" label="A label">
+  <option disabled selected value="">Pick something</option>
+  <option value="foo">Foo</option>
+</w-select>
+```
+
+## Props
+All typical HTML5 attributes are valid props for select, see Field for additional props.
```
Comment (BalbinaK) [inline docs/select/vue.md]: Now that we have the possibility to link to the `Field` props, maybe we could do it here? Or we could even display the props table for `field` component directly here. Wdyt?
```
<api-table type=vue component="Field"/>
```

**Review by BalbinaK — COMMENTED** [review]
Cool stuff! Just a few small comments :)

File: docs/select/Example.vue (hunk 1 above)
Comment (AnnaRybkina) [reply docs/select/Example.vue:9]: Not really 😁 Updated https://github.com/warp-ds/tech-docs/pull/21/commits/70b54696e43529bc7393807b0985a25d198a5983

File: docs/select/react.md (hunk 2 above)
Comment (AnnaRybkina) [reply docs/select/react.md]: Good point, updated everywhere https://github.com/warp-ds/tech-docs/pull/21/commits/70b54696e43529bc7393807b0985a25d198a5983

File: docs/select/vue.md (hunk 3 above)
Comment (AnnaRybkina) [reply docs/select/vue.md]: 🤦 tnx

Comment (AnnaRybkina) [reply docs/select/vue.md]: https://github.com/warp-ds/tech-docs/pull/21/commits/70b54696e43529bc7393807b0985a25d198a5983

File: docs/select/vue.md (hunk 4 above)
Comment (AnnaRybkina) [reply docs/select/vue.md]: https://github.com/warp-ds/tech-docs/pull/21/commits/70b54696e43529bc7393807b0985a25d198a5983

File: docs/select/vue.md (hunk 5 above)
Comment (AnnaRybkina) [reply docs/select/vue.md]: Yes, agreed. https://github.com/warp-ds/tech-docs/pull/21/commits/70b54696e43529bc7393807b0985a25d198a5983

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**